components/TopNav.vue

```diff
@@ -1,16 +1,16 @@
 <template>
-    <div class="p-4 pl-8 border-b-2 border-secondary-bg/40 flex justify-between items-center">
+    <div class="p-4 pl-8 border border-secondary-bg/20 flex justify-between items-center bg-gradient-to-t from-secondary-bg/10 via-primary-bg to-primary-bg">
```

Comment:
Can you provide a screenshot for what this is supposed to look like? I don't see it in the preview.

Reply:
I bumped it up to make it more noticeable. It shows completely different across my  laptop and desktop screens. 

It's more noticeable on any theme other than `theme-coral`.

`theme-neon`
<img width="1387" alt="Screenshot 2023-01-08 at 9 30 56 PM" src="https://user-images.githubusercontent.com/4411499/211196540-bf9dc545-b9eb-4347-9c4c-b0356eb923ea.png">

`theme-ocean`
<img width="1386" alt="Screenshot 2023-01-08 at 9 41 45 PM" src="https://user-images.githubusercontent.com/4411499/211196570-b4d4c055-8a00-4d5e-830f-ea190939e6ec.png">

Reply:
Why didn't you call it theme chocomint? 🌿

Reply:
Just to confirm, that glassy effect in the selectors... is that from your browser? Those selectors look different in Chrome. I also notice a slight gap in between the border and the selector in your screenshot. 

<img width="743" alt="localhost_3000" src="https://user-images.githubusercontent.com/4602369/211197753-c8504988-b0fd-46e0-bcfa-8b887c11adea.png">

Reply:
yeah, it's different across browsers. That's why I've got a separate issue #147 for polishing them up.